website/versioned_docs/version-V1/advanced/caching.mdx

```diff
@@ -9,13 +9,13 @@ There are 3 different ways (polling modes) to control caching.
 ## Auto polling (default)
 
 In auto polling mode, the ConfigCat SDK downloads the latest values automatically and stores them in the cache.
-This is done in every 60 seconds by default.
-You can set the polling interval to any number between 1 second and int max.
+By default, this happens every 60 seconds.
+You can set the polling interval to any value between 1 second and the maximum integer limit.
```

Comment:
As per the [task description of the Docs maintenance](https://trello.com/c/HsyThkyC) week, V1 docs shouldn't be updated except for broken links and code. Please revert changes other than those.